services/src/main/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionService.java

```diff
@@ -357,9 +357,6 @@ private void handleSoapFaultAndWebServiceException(final IIndividualRequest indi
 
         final String targetAppCode =
                 individualRequest.getBulkSubmission().getTargetApplication().getTargetApplicationCode();
```

Comment:
Remove this, the code above that sets up the messageObj variable and the "Create a new message for DLQ".  Removing the code to add the message to the DLQ means they are no longer needed.

Reply:
Redundant code has been removed.